### PR TITLE
feat: COR-1547 archived National and Municipalities Positive tests pages

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard27**
## Summary
# Done
- Place on National and Municipalities _"Positive Tests"_ pages in the archived section in [nl-layout.tsx](https://github.com/minvws/nl-covid19-data-dashboard/compare/feature/COR-1547-archiving-positive-tests-page?expand=1#diff-28e390aba8b17946ab0b7651888ae13110a8655693631325907494237223eac8) and [gm-layout.tsx](https://github.com/minvws/nl-covid19-data-dashboard/compare/feature/COR-1547-archiving-positive-tests-page?expand=1#diff-099a92d67d995d8dfcb0c40af083d67b55ab8f93ff9c597b3fb06dbd229c65f1).
- Change sub-title of "Positive Tests"  page _"Ontwikkeling van het virus"_ => _**"Gearchiveerde pagina's"**_  on National and Municipalities levels.
- Added right under the introduction of the page - a grey notification bar, to indicate the page is archived on National and Municipalities levels.
- Remove first graph _"Zelfgerapporteerde positieve coronatestuitslagen"_ from the archived page, as this graph will still be displayed at the new _"Testen"_ page on National level.
- Removed the _"Gearchiveerde grafieken"_ section with button and  kept the _"Positieve testen groeigetal"_   graph that appeared after clicking. 
- VR connections was removed from NL [positief-geteste-mensen.tsx](https://github.com/minvws/nl-covid19-data-dashboard/compare/feature/COR-1547-archiving-positive-tests-page?expand=1#diff-8e7646c3094376fe95f9e858441ab57b1b626ae513deadc0e9fa0c70d2651006)

## In edition
- **Grey notification bars** on NL level at _"Positieve testen groeigetal"_ and _"Positieve GGD tests over time"_ sections can be removed by communication. 

## Screenshots
**_Before_**
<details>
  <summary>Toggle before screenshot</summary> 

Screenshot 2023-04-19 at 11 29 29
Screenshot 2023-04-19 at 11 30 49

</details>

**_After_**
<details>
  <summary>Toggle after screenshot</summary> 

Screenshot 2023-04-19 at 11 27 33
Screenshot 2023-04-19 at 11 31 36

</details>

### CODE REVIEW

File: packages/app/schema/gm/tested_overall_archived_20230417.json [hunk 1]
```diff
@@ -29,17 +32,15 @@
...
-        },
         "date_of_insertion_unix": {
           "type": "integer"
         }
       }
     }
   },
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "title": "nl_tested_overall",
+  "$id": "tested_overall_archived_20230417.json",
```
Comment (VWSCoronaDashboard28) [inline packages/app/schema/gm/tested_overall_archived_20230417.json:42]: Why is the `$id` needed? I feel like I haven't seen this added before. I could be wrong, but still curious what this does.

File: packages/app/schema/nl/tested_overall_archived_20230417.json [hunk 2]
```diff
@@ -32,15 +29,17 @@
...
         "date_of_insertion_unix": {
           "type": "integer"
         }
       }
     }
   },
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "$id": "tested_overall.json",
-  "title": "gm_tested_overall",
+  "title": "nl_tested_overall_archived_20230417",
```
Comment (VWSCoronaDashboard28) [inline packages/app/schema/nl/tested_overall_archived_20230417.json:42]: In this case the `$id` that used to be there is now removed 🤔 Is this supposed to happen?

File: packages/app/src/components/chart-tile-toggle.tsx [hunk 3]
```diff
@@ -14,7 +14,7 @@ export interface ChartTileToggleProps {
 
 export const ChartTileToggle = ({ initialValue, items, onChange }: ChartTileToggleProps) => {
   return (
-    <Box display="flex">
+    <Box display="flex" style={{ marginTop: '24px', marginBottom: '24px' }}>
```
Comment (VWSCoronaDashboard28) [inline packages/app/src/components/chart-tile-toggle.tsx:17]: The style here can make use of CSS shorthand properties. You could choose between:
1. `style={{ margin: '24px' 0 }}`
2. `style={{ marginBlock: '24px'}}`

Additionally, is there a specific reason for why you chose adding an inline style on the `Box`? If not then I would suggest simply adding `margin="24px 0"` to the  `Box` component here.

File: packages/app/src/pages/landelijk/positief-geteste-mensen.tsx [hunk 4]
```diff
@@ -95,8 +92,8 @@ function PositivelyTestedPeople(props: StaticProps<typeof getStaticProps>) {
   const [confirmedCasesInfectedPerAgeTimeframe, setConfirmedCasesInfectedPerAgeTimeframe] = useState<TimeframeOption>(TimeframeOption.ALL);
 
   const { commonTexts, formatNumber, formatDateFromSeconds } = useIntl();
+
```
Comment (VWSCoronaDashboard28) [inline packages/app/src/pages/landelijk/positief-geteste-mensen.tsx:95]: I think this empty line could be removed?

**Review by VWSCoronaDashboard28 — COMMENTED** [review]
Please see my comments

**Review by VWSCoronaDashboard28 — COMMENTED** [review]
I also noticed that the Typecheck action fails. Something to do with the warning text. Could you double check whats happening there by running `yarn typecheck` locally?

File: packages/cms/src/lokalize/key-mutations.csv [hunk 5]
```diff
@@ -394,6 +394,8 @@ timestamp,action,key,document_id,move_to
 2023-03-21T10:35:31.653Z,delete,pages.vaccinations_page.shared.vaccination_grade_tile.autumn_labels.description_footer,u6z14vWwH5lzTVbDpRg06r,__
 2023-03-21T10:35:31.653Z,delete,pages.vaccinations_page.shared.vaccination_grade_tile.fully_vaccinated_labels.description_footer,772MCeNIooHnNdtyUIs7TE,__
 2023-03-21T10:35:31.653Z,delete,pages.vaccinations_page.vr.vaccination_grade_toggle_tile.description_footer,qjZwg5QUJcN8JoCoxxDHxs,__
+2023-04-13T07:22:16.246Z,add,pages.positive_tests_page.shared.belangrijk_bericht,mt9hMqvnezW3fDtFvfvcu2,__
+2023-04-13T07:26:08.198Z,delete,pages.positive_tests_page.shared.belangrijk_bericht,mt9hMqvnezW3fDtFvfvcu2,__
```
Comment (VWSCoronaDashboard26) [inline packages/cms/src/lokalize/key-mutations.csv:398]: Is the addition and deletion of the same key intentional?

**Review by VWSCoronaDashboard26 — CHANGES_REQUESTED** [review]

File: packages/app/schema/gm/__static_values_archived_20230417.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "title": "gm_static_values",
+  "title": "gm_static_values_archived_20230417",
```
[the remark this answers is not in the document]
Comment (VWSCoronaDashboard27) [reply packages/app/schema/gm/__static_values_archived_20230417.json]: Done

File: packages/cms/src/lokalize/key-mutations.csv (hunk 5 above)
Comment (VWSCoronaDashboard27) [reply packages/cms/src/lokalize/key-mutations.csv:398]: Nope, probably I made a mistake which I fixed later.

File: packages/app/src/pages/landelijk/positief-geteste-mensen.tsx (hunk 4 above)
Comment (VWSCoronaDashboard27) [reply packages/app/src/pages/landelijk/positief-geteste-mensen.tsx:95]: done

File: packages/app/schema/gm/__index.json
[the remark this answers is not in the document]
Comment (VWSCoronaDashboard27) [reply packages/app/schema/gm/__index.json:1]: Changes have been reverted.

File: packages/app/schema/gm/tested_overall_archived_20230417.json [hunk 7]
```diff
@@ -1,19 +1,22 @@
 {
   "definitions": {
     "value": {
-      "title": "nl_tested_overall_value",
+      "title": "gm_tested_overall_archived_20230417_value",
```
[the remark this answers is not in the document]
Comment (VWSCoronaDashboard27) [reply packages/app/schema/gm/tested_overall_archived_20230417.json:4]: Changes have been reverted

File: packages/app/schema/gm/tested_overall_archived_20230417.json (hunk 1 above)
Comment (VWSCoronaDashboard27) [reply packages/app/schema/gm/tested_overall_archived_20230417.json:42]: Changes have been reverted.

File: packages/app/schema/gm/__difference.json [hunk 8]
```diff
@@ -3,10 +3,10 @@
   "title": "gm_difference",
   "type": "object",
   "properties": {
-    "tested_overall__infected_per_100k_moving_average": {
+    "tested_overall__infected_per_100k_moving_average_archived_20230417": {
```
Comment (VWSCoronaDashboard18) [inline packages/app/schema/gm/__difference.json:6]: Not to be nitpicky, and it's definitely non-blocking. But we usually use the date we go live with archiving as the date suffix. And this seems to be the date the code is changed.

But as I said, this is definitely nonblocking and feel free to resolve it by yourself.

File: packages/app/src/pages/gemeente/[code]/positief-geteste-mensen.tsx [hunk 9]
```diff
@@ -65,7 +66,7 @@ export const getStaticProps = createGetStaticProps(
       const { locale } = context;
       return `{
         "parts": ${getPagePartsQuery('positive_tests_page')},
-        "elements": ${getElementsQuery('gm', ['tested_overall'], locale)}
+        "elements": ${getElementsQuery('gm', ['tested_overall_archived_20230417'], locale)}
```
Comment (VWSCoronaDashboard18) [inline packages/app/src/pages/gemeente/[code]/positief-geteste-mensen.tsx:69]: These elements currently do not exist in sanity. Are the _archived elements added? Or was it unintentional? See at @VWSCoronaDashboard26 's comment about this and the timelines shared in the standup last week.

File: packages/app/src/pages/landelijk/positief-geteste-mensen.tsx [hunk 10]
```diff
@@ -124,12 +120,14 @@ function PositivelyTestedPeople(props: StaticProps<typeof getStaticProps>) {
 
   const lastInsertionDateOfPage = getLastInsertionDateOfPage(data, pageMetrics);
 
+  const hasActiveWarningTile = !!textShared.belangrijk_bericht;
```
Comment (VWSCoronaDashboard18) [inline packages/app/src/pages/landelijk/positief-geteste-mensen.tsx:123]: `textShared.belangrijk_bericht` doesn't exist in sanity, making it impossible to show the warning.

File: packages/app/src/components/chart-tile-toggle.tsx (hunk 3 above)
Comment (VWSCoronaDashboard27) [reply packages/app/src/components/chart-tile-toggle.tsx]: @VWSCoronaDashboard28  The style component used here to be able to overwrite margin only for this specific toggle.

**Review by VWSCoronaDashboard26 — APPROVED** [review]

File: packages/app/src/components/chart-tile-toggle.tsx (hunk 3 above)
Comment (VWSCoronaDashboard18) [reply packages/app/src/components/chart-tile-toggle.tsx]: I don't see this comment solved or adjusted to use the `space[2]` value.

File: packages/app/src/pages/gemeente/[code]/positief-geteste-mensen.tsx [hunk 11]
```diff
@@ -99,13 +100,14 @@ function PositivelyTestedPeople(props: StaticProps<typeof getStaticProps>) {
   };
 
   const lastInsertionDateOfPage = getLastInsertionDateOfPage(data, pageMetrics);
+  const hasActiveWarningTile = !!textShared.warning;
```
Comment (VWSCoronaDashboard28) [inline packages/app/src/pages/gemeente/[code]/positief-geteste-mensen.tsx:103]: Nitpicky/non-blocking, but this is a very small.short condition, I don't see the need to abstract/extract `!!textShared.warning` to the `hasActiveWarningTile` variable. 

You could just:
```
{!!textShared.warning && <WarningTile isFullWidth message={textShared.warning} variant="informational" />}
```

File: packages/app/src/pages/landelijk/positief-geteste-mensen.tsx [hunk 12]
```diff
@@ -122,12 +120,14 @@ function PositivelyTestedPeople(props: StaticProps<typeof getStaticProps>) {
 
   const lastInsertionDateOfPage = getLastInsertionDateOfPage(data, pageMetrics);
 
+  const hasActiveWarningTile = !!textShared.warning;
```
Comment (VWSCoronaDashboard28) [inline packages/app/src/pages/landelijk/positief-geteste-mensen.tsx:123]: Same comment as before, seems unnecessary to extract this condition to its own variable.

File: packages/app/src/components/chart-tile-toggle.tsx (hunk 3 above)
Comment (VWSCoronaDashboard28) [reply packages/app/src/components/chart-tile-toggle.tsx]: I'm slightly confused as the file on which my comments were made is no longer a part of this PR?

**Review by VWSCoronaDashboard28 — APPROVED** [review]
Two small additional comments

File: packages/app/src/components/chart-tile-toggle.tsx (hunk 3 above)
Comment (VWSCoronaDashboard27) [reply packages/app/src/components/chart-tile-toggle.tsx]: @VWSCoronaDashboard28  After discussion with design changes have been reverted back to 16px.

**Review by VWSCoronaDashboard25 — APPROVED** [review]
Looks good to me!

File: packages/app/src/pages/gemeente/[code]/positief-geteste-mensen.tsx (hunk 11 above)
Comment (VWSCoronaDashboard27) [reply packages/app/src/pages/gemeente/[code]/positief-geteste-mensen.tsx]: done.

**Review by VWSCoronaDashboard26 — APPROVED** [review]

**Review by VWSCoronaDashboard18 — APPROVED** [review]
Nice! Let's go!

**Pull request merged**